mono/eglib/gutf8.c

```diff
@@ -291,7 +291,7 @@ g_utf8_find_prev_char (const gchar *str, const gchar *p)
 {
 	while (p > str) {
 		p--;
-		if ((*p & 0xc0) != 0xb0)
+		if ((*p & 0xc0) != 0x80)
```

Comment:
Please consider deleting the function. The rationale should say that it is incorrect and unused.